index.rst

```diff
@@ -249,6 +344,11 @@ What are the open questions that drive the differences between different viable
 
 - Can we feed FGCM with compensated-filter photometry in order to run it before background estimation is complete?
 
+- Do we need some kind of coaddition or warp-comparison to finalize our per-visit background models?  If so, will those also impact PSF modeling and/or aperture corrections?
+
+- Is a second round of astrometric fitting necessary to ensure PSF chromaticity is consistent with our (achromatic) WCSs?  Is it sufficient?
+
+- Are there small changes we could make to better share code with AP?
```

Comment:
I believe `CalibrateImageTask` is the primary sharing point: so long as the inputs to that are fluence images with decent camera geometry models that we can fit an affine WCS on top of, the `initial_pvi` should be decent.

After that, it would be great if we could easily make a "get good coadds and enough information to vet them with DRP-like tools" pipeline (our current template generation pipeline is not as sophisticated as DRP's coadd building). Obviously, we want to share as much of the image subtraction code as possible, but I think that's going to happen anyway.

Reply:
I agree and I'll go ahead and delete the bullet point.

As for coadds, aside from Y1 _incremental_ templates I think it's probably a subset of a real DRP pipeline you want, not a new DRP-like pipeline.  And while it's not something we have wired together yet, I think eventually you should see that pipeline as producing a deep reference catalog for AP to consume as well as templates (including - but not only - to get object associations for alerts).  That means going a bit further than coaddition, but there's probably still enough in the DRP pipelines after that that a subset would still be worthwhile.